lib/stimulus_reflex/logger.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+module StimulusReflex
+  class Logger
+    attr_accessor :reflex, :current_operation
+
+    def initialize(reflex)
+      @reflex = reflex
+      @current_operation = 1
+    end
+
+    def print
+      return unless config_logging.lambda?
+      puts
+      reflex.broadcaster.operations.each do
+        puts config_logging.call(self) + "\e[0m"
+        @current_operation += 1
+      end
+      puts
+    end
+
+    private
+
+    def config_logging
+      return @config_logging if @config_logging
```

Comment:
@RolandStuder Where does this instance variable come from?

Reply:
Is this the one defined in line 28? I am having difficulties with grasping what's happening here.

Reply:
Yes, line 28! It's memoizing to @config_logging

Reply:
Ah `@config_logging` is nil in a first run but then it gets assigned an actual value. I see now. Thanks.